backend/go.mod

```diff
@@ -1,63 +1,63 @@
 module github.com/cilium/hubble-ui/backend
 
-go 1.21
+go 1.21.6
```

Comment:
Patch component of version is not supported here afaik, thus codeql is crashing

Reply:
I don't understand why it's working there https://github.com/cilium/hubble-ui/pull/727

Reply:
Patch version is supported here, can't figure out in what case codeql crashes